### PR TITLE
Add dynamic entities directive

### PR DESCRIPTION
**Pull request by wuelcas**
Fix #264

### CODE REVIEW

**Comment by patcat** [comment]
Oh neat! Is this a more inbuilt way to do what I was doing in https://github.com/VoxaAI/voxa/issues/253?

**Comment by wuelcas** [comment]
That's right @patcat

File: src/platforms/alexa/directives.ts [hunk 1]
```diff
@@ -671,3 +671,35 @@ export class VideoAppLaunch extends MultimediaAlexaDirective {
...
+  }
+
+  public async writeToReply(
+    reply: IVoxaReply,
+    event: IVoxaEvent,
+    transition?: ITransition,
+  ): Promise<void> {
+    let types = [];
+
+    if (this.viewPath) {
```
[the remark this answers is not in the document]
Comment (wuelcas) [reply src/platforms/alexa/directives.ts]: Sorry, copy and pasted it from another directive

File: docs/alexa-directives.rst [hunk 2]
```diff
@@ -421,3 +421,50 @@ The only required parameter is the ``slotToElicit``, but you can also pass in th
         to: "someOtherThing",
       };
     });
+
```
[the remark this answers is not in the document]
Comment (wuelcas) [reply docs/alexa-directives.rst]: I agree, I'll make the change

[the remark this answers is not in the document]
Comment (wuelcas) [reply docs/alexa-directives.rst]: I added support for the types array and the whole directive, check it out and let me know what you think.

**Pull request merged**